Vagrantfile

```diff
@@ -77,6 +77,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     agent.vm.provision :shell, :inline => "facter --custom-dir=/vagrant/lib/facter sensu_version"
   end
 
+  config.vm.define "ubuntu1804-agent", autostart: false do |agent|
```

Comment:
```
==> ubuntu1804-agent: Running provisioner: shell...
    ubuntu1804-agent: Running: inline script
    ubuntu1804-agent: /tmp/vagrant-shell: line 1: puppet: command not found
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

Reply:
The docker based tests work.  I ran into several issues with Ubuntu 18.04 and vagrant / virtual box.  One such issue was ifup/ifdown not being present and breaking some of the workflows needed by vagrant.  The issue about puppet missing is something else, will look into it.

Reply:
Arg, I forgot to commit change to vagrant boxe provisioning that used puppet5.  Ubuntu 18.04 is only supported by very latest puppet 5.